cypress/e2e/Courses/Certificates/LifetimePlanAwardedCourseCertificate.feature

```diff
@@ -1,10 +1,14 @@
+@course @certificate @lifetime-plan @business:low-impact
```

Comment:
Of course, downloading the certificate is important, but for the business is more important to focus on the unawarded ones (potential buyers). Another option could be to increase the business importance and add low-likely